docs/source/app-dev/grpc/index.rst

```diff
@@ -7,11 +7,6 @@ gRPC
 ####
 
 
-.. toctree::
-   :hidden:
-
-   error-codes
```

Comment:
FYI @tudor-da Had to move it to the bottom of the file because for pdfs the toctree items are replaced by the content of the referenced file which resulted in the subsections being earlier in the pdf the the parent section.

BTW. I think I spotted similar inversion problem in at least one other part of the docs. Will look into it.